metal/resource_metal_spot_market_request_test.go

```diff
@@ -26,9 +26,9 @@ data "metal_spot_market_request" "dreq" {
 }
 
 resource "metal_spot_market_request" "request" {
-  project_id       = "${metal_project.test.id}"
+  project_id       = metal_project.test.id
   max_bid_price    = data.metal_spot_market_price.test.price * 1.2
-  facilities       = ["ewr1"]
+  facilities       = ["sv15"]
```

Comment:
In the future, maybe we can include data sources with these tests to determine which facility will be the best for the test: https://github.com/equinix/terraform-provider-metal/issues/164#issuecomment-884896686.  This would make testing dependent on an additional resource and API call, but it should help pass more tests overall.

(or perhaps we can make a single capacity API check at the start of the tests and use that facility in all tests? )